dist/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wbpack Exercise</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <section>
+      <div class="general-wrapper">
+        <div class="top-container">
+          <header>
+            <h1 id="main-title">Leaderboard</h1>
+          </header>
+          <!-- display section of the score-board -->
+          <div class="score-bored-heading">
+            <div class="score-board">
+              <h2>Recent scores</h2>
+              <button type="button" class="btn refresh-btn">Refresh</button>
+            </div>
+            <div class="display-score">
+              <ul class="score-list" id="score">
+                <li>AZEEM ADE : 29</li>
+                <li>AZEEM BAYO : 99</li>
+                <li>AZEEM ADEBAYO: 89</li>
+                <li>AZEEM ADISA: 36</li>
```

Comment:
- [ ] good job creating the layout with HTML 👍  however according to the requirements HTML should be used just to make the layout, to fill in the list you should use ES6 modules, with import and export.

Reply:
Noted,
Thank you for your observation.